proto/api/v1/cluster_service.proto

```diff
@@ -23,8 +23,19 @@ enum LoadBalancerType {
     ROUTE         = 3;
 }
 
+// next available tag: 3
+message DecommissionedClusterRetentionInfo {
+    oneof RetentionInfo {
+        // indicates whether a cluster is protected from deletion
+        bool is_protected_cluster    = 1;
+        // days after which cluster will be deleted if sensor health remains UNHEALTHY
+        int32 days_until_deletion  = 2;
```

Comment:
Forgive me for not reading code to answer this question: what is value if chluser

Reply:
@pedrottimark This will be embedded into `ClusterResponse`, and cluster response will already have the cluster. 
```
message ClusterResponse {
     storage.Cluster cluster                                                                 = 1;
     DecommissionedClusterRetentionInfo cluster_retention_info  = 2;
 }
```

In case of `CluserList` , the ClusterList will have a map<clusterID -> DecommissionedClusterRetentionInfo > that will have entries for all `SensorStatus UNHEALTHY` clusters.
```
message ClustersList {
     repeated storage.Cluster clusters                                                                                 = 1;
     // Maps 'UNHEALTHY' clusters' IDs to their retention info
     map<string, DecommissionedClusterRetentionInfo> cluster_id_to_retention_info = 2;
 }
```